CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
Is the list exhaustive (at least w.r.t explicitly reported bugs)? Matthieu said there were more than 450 bugs fixed.

Maybe "Excerpt of bug fixes" or "Selected list of bug fixes", depending on your intention.